stdlib/list.gr

```diff
@@ -195,3 +195,25 @@ let unique = (list) => {
   }
   iter(list, [])
 }
+
+let rec drop = (n, list) => {
+  if (n < 0) {
+    fail 'number of items to drop cannot be a negative number'
+  } else {
+    match ((n, list)) {
+      | (_,[]) => []
+      | (n, _) when n == 0 => list
```

Comment:
Looks like we can't pattern match on number so I used a guard here.

Reply:
@ospencer Is this a bug or a TODO?

Reply:
We already have an issue: https://github.com/grain-lang/grain/issues/284